deployments/notifyRewardAmount/xDAI/fox_regen_distribute.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-use-before-define */
+const hre = require("hardhat");
+const { ethers } = hre;
+
+const pools = [
+    {
+        address: "0x502EC7a040F486EE6Cb7d634D94764874B29dE68",
```

Comment:
Fun fact: this proxy's implementation (0x5f672d71399d8cDbA64f596394b4f4381247E025, UnipoolTokenDistributor) has been verified on blockscout with flattened source code which is built from the wrong version of OpenZeppelin. (This repo specifies 4.1.0 but the verified code uses something above 4.3.) This gave me a bit of a fright, but I tracked down every single OZ change between the two and confirmed that they didn't affect the bytecode output. (Some changes were just cosmetic, but the SafeMath average and ceilDiv routines actually use different code; luckily they aren't used in the actual contract anywhere.)